gnome-base/nautilus/nautilus-46.2.ebuild

```diff
@@ -0,0 +1,121 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson readme.gentoo-r1 virtualx xdg
+
+DESCRIPTION="Default file manager for the GNOME desktop"
+HOMEPAGE="https://apps.gnome.org/Nautilus/"
+
+LICENSE="GPL-3+ LGPL-2.1+"
+SLOT="0"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+
+IUSE="+cloudproviders gnome +gstreamer gtk-doc +introspection +previewer selinux"
+REQUIRED_USE="gtk-doc? ( introspection )"
```

Comment:
As explained at 
https://gitlab.gnome.org/GNOME/nautilus/-/issues/3150

We should rename gtk-doc USE to doc and handle docs normally

This should also Close: https://bugs.gentoo.org/911997

Reply:
Done in https://github.com/gentoo/gentoo/pull/36834/commits/78213b281a9371b220b4ec0969107c89f1c770ec